userspace/libraries/libxml/Declaration.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+#include <libmath/Vec2.h>
+#include <libutils/Vector.h>
+
+namespace Xml
+{
+using Version = Math::Vec2i;
+class Declaration
+{
+private:
+    Version _version = {1, 0};
+    String _encoding = "UTF-8";
+    bool _standalone = true;
+
+public:
```

Comment:
No getters ?

Reply:
I don't set anything for the declaration yet. It's "handled/skipped" but not parsed yet